Query/README.md

```diff
@@ -34,6 +34,23 @@ returns:
 	status: uint8 - one of the `QueryStatus` variants. See `Query.sol` for details.
     objects: JsonLib.Value[] - array of json values. Every json value represents fields requested in `returnFilter` for certain `CollectionType` variant: message, transaction, account.
 
+`waitForCollection` - returns an object that fulfills the conditions or waits for its appearance.
+
+arguments:
+
+    answerId: uint32 - id of function callback.
+    collectionType: uint8 - one of the `CollectionType` variants: 
+        Accounts (0), Messages (1), Transactions(2). 
+        See Query.sol for details.
+    queryFilter: string - utf-8 string with collection filter.
+    returnFilter: string - utf-8 string with projection (result) string.
+    timeout: uint32 - query timeout in ms.
```

Comment:
In milliseconds? uint32 it too small for this.

Reply:
https://github.com/tonlabs/TON-SDK/blob/88ecc118516fffb710522e4a2a4e17c7df7c94a8/ton_client/src/net/queries.rs#L116
it is timeout. So we assume that you don't want wait for years.. =)